src/components/Header.jsx

```diff
@@ -31,30 +32,39 @@ class Header extends Component {
     });
   }
 
-  handleKeyPress = (event) => {
+  handleKeyDown = (event) => {
     if (event.keyCode === 13 && event.target.value !== "") {
       this.handleSearch();
     }
   }
 
   render() {
+    const titleClass = classNames({
+      "red lighten-1 btn buttons": this.state.searchBy === "title",
+      "grey lighten-1 btn buttons": this.state.searchBy === "genre"
+    });
+
+    const genreClass = classNames({
+      "red lighten-1 btn buttons": this.state.searchBy === "genre",
+      "grey lighten-1 btn buttons": this.state.searchBy === "title"
+    });
+
```

Comment:
I'd suggest using different name (than red/grey) to indicate that given search is active.

```js
const { searchBy } = this.state;

const titleClass = classNames('lighten-1 btn buttons', {
  'active': this.state.searchBy === "title",
});

const genreClass = classNames('lighten-1 btn buttons', {
  'active': this.state.searchBy === "genre",
});
```